client/app/states/login/login.state.js

```diff
@@ -36,6 +36,10 @@
       password: API_PASSWORD,
     };
 
+    if ($window.location.href.includes("?timeout")) {
+      Notifications.message('danger', '', __('Your session has timed out.'), true);
```

Comment:
`EventNotifications.error(__('Your session has timed out.'));`

should be used here, while you're touching this file might as well remove the other use of `Notifications` (depreciated in favor of `EventNotifications`)